powered-by.html

```diff
@@ -808,6 +813,22 @@ <h3>Want to appear on this page?</h3>
 
 <script>
     $(function () {
+        // enforce alphabetical sort by root domain for fairness
```

Comment:
I wonder if we are running this once per page load anyway if it is fair to randomize the logos. 

That could be considered more fair. Not sure if folks expect the same ordering on reload.

Reply:
I looked at the list as being mostly alphabetical and accepted that as being the original intent. Would random order upset any search bots?

Reply:
That's fair. Perhaps this could be a separate discussion from this PR. 👍